[2024-03-30T10:00:02.992+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_Flow.insert_data_from_files scheduled__2024-03-25T21:11:00+00:00 [queued]>
[2024-03-30T10:00:03.003+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_Flow.insert_data_from_files scheduled__2024-03-25T21:11:00+00:00 [queued]>
[2024-03-30T10:00:03.004+0000] {taskinstance.py:2193} INFO - Starting attempt 2 of 7
[2024-03-30T10:00:03.023+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): insert_data_from_files> on 2024-03-25 21:11:00+00:00
[2024-03-30T10:00:03.029+0000] {standard_task_runner.py:60} INFO - Started process 14260 to run task
[2024-03-30T10:00:03.033+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ETL_Flow', 'insert_data_from_files', 'scheduled__2024-03-25T21:11:00+00:00', '--job-id', '2364', '--raw', '--subdir', 'DAGS_FOLDER/ETL.py', '--cfg-path', '/tmp/tmpbspa4pu2']
[2024-03-30T10:00:03.037+0000] {standard_task_runner.py:88} INFO - Job 2364: Subtask insert_data_from_files
[2024-03-30T10:00:03.100+0000] {task_command.py:423} INFO - Running <TaskInstance: ETL_Flow.insert_data_from_files scheduled__2024-03-25T21:11:00+00:00 [running]> on host cb216a3a40e5
[2024-03-30T10:00:03.430+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='ETL_Flow' AIRFLOW_CTX_TASK_ID='insert_data_from_files' AIRFLOW_CTX_EXECUTION_DATE='2024-03-25T21:11:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-25T21:11:00+00:00'
[2024-03-30T10:00:03.894+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-03-30T10:00:04.107+0000] {sql.py:572} INFO - Done loading. Loaded a total of 92 rows into MRR_Dim_Customers
[2024-03-30T10:00:04.173+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-03-30T10:00:04.305+0000] {sql.py:572} INFO - Done loading. Loaded a total of 47 rows into MRR_Dim_Employees
[2024-03-30T10:00:04.376+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-03-30T10:00:04.579+0000] {sql.py:572} INFO - Done loading. Loaded a total of 77 rows into MRR_Dim_Products
[2024-03-30T10:00:04.696+0000] {taskinstance.py:2728} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/data_insertion.py", line 38, in insert_data_from_files
    query = """
KeyError: 'df'
[2024-03-30T10:00:04.711+0000] {taskinstance.py:1149} INFO - Marking task as UP_FOR_RETRY. dag_id=ETL_Flow, task_id=insert_data_from_files, execution_date=20240325T211100, start_date=20240330T100002, end_date=20240330T100004
[2024-03-30T10:00:04.727+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 2364 for task insert_data_from_files ('df'; 14260)
[2024-03-30T10:00:04.774+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-03-30T10:00:04.814+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/models/baseoperator.py:1201: AirflowProviderDeprecationWarning: Call to deprecated class PostgresOperator. (Please use `***.providers.common.sql.operators.sql.SQLExecuteQueryOperator`.Also, you can provide `hook_params={'schema': <database>}`.)
  result = cls.__new__(cls)

[2024-03-30T10:00:04.841+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
